uninstall.php

```diff
@@ -1,51 +1,48 @@
 <?php
-
-// If uninstall not called from WordPress exit
 defined( 'WP_UNINSTALL_PLUGIN' ) or die( 'Cheatin&#8217; uh?' );
 
-// Delete all transients
+// Delete all transients.
 delete_site_transient( 'update_wprocket' );
 delete_site_transient( 'update_wprocket_response' );
 delete_transient( 'wp_rocket_settings' );
 delete_transient( 'rocket_check_licence_30' );
 delete_transient( 'rocket_check_licence_1' );
 delete_transient( 'rocket_cloudflare_ips' );
 
-// Delete WP Rocket options
+// Delete WP Rocket options.
 delete_option( 'wp_rocket_settings' );
 
-// Delete all user meta related to WP Rocket
+// Delete all user meta related to WP Rocket.
 delete_metadata( 'user', '', 'rocket_boxes', '', true );
 
-// Clear scheduled WP Rocket Cron
+// Clear scheduled WP Rocket Cron.
 wp_clear_scheduled_hook( 'rocket_purge_time_event' );
 wp_clear_scheduled_hook( 'rocket_database_optimization_time_event' );
 
 /**
- * Remove all cache files
+ * Remove all cache files.
  *
  * @since 1.2.0
  *
- * @param string $dir Directory path to remove.
+ * @param string $dir Path to directory.
  */
-function __rocket_rrmdir( $dir ) {
+function rocket_uninstall_rrmdir( $dir ) {
 
 	if ( ! is_dir( $dir ) ) {
-		@unlink( $dir );
-		return;	
+		rocket_direct_filesystem()->delete( $dir );
```

Comment:
You're right. So we have two solutions:

- Revert to unlink
- Duplicate the function for the filesystem to keep with the coding standards

Which do you prefer?